stargz/fs.go

```diff
@@ -279,6 +281,10 @@ func (fs *filesystem) Mount(ctx context.Context, mountpoint string, labels map[s
 		// necessary for layer verification.
 		l.skipVerify()
 		log.G(ctx).Warningf("No verification is held for layer")
+	} else if fs.disableVerification {
+		// Verification is disabled completely
+		l.skipVerify()
+		log.G(ctx).Debugf("Verification forcefully skipped")
```

Comment:
`disable_verification` disables verification completely, so this check should come to the top of the `if` chain and should disable the verification even for eStargz images.

Reply:
Ok, moved to the top.